base/strings/basic.jl

```diff
@@ -34,6 +34,7 @@ If you need to subsequently modify `v`, use `String(copy(v))` instead.
 """
 String(v::Array{UInt8,1})
 
+String(x::Symbol) = string(x)
```

Comment:
Should be more efficient to do `String(x::Symbol) = unsafe_string(Cstring(x))`, which copies directly from the underlying data.  (Seems to be about 3x faster in a quick benchmark.)